example/ndpiReader.c

```diff
@@ -1269,7 +1269,6 @@ static void parseOptions(int argc, char **argv) {
 
     case 'T':
       max_num_tcp_dissected_pkts = atoi(optarg);
-      if(max_num_tcp_dissected_pkts < 3) max_num_tcp_dissected_pkts = 3;
```

Comment:
Does it make sense for TCP to allow dissecting less than 3 packets if `SYN` and `ACK` packets also count?

Reply:
Probably not. Something like `if(max_num_tcp_dissected_pkts < 3 && max_num_tcp_dissected_pkts != 0) max_num_tcp_dissected_pkts = 3;`?

Reply:
Let's keep the same logic; it is only an example showing what any real application could/might do